packages/pages/package.json

```diff
@@ -82,9 +83,13 @@
     "vitepress": "1.0.0-draft.8"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-typescript": "^7.16.7",
     "@microsoft/api-documenter": "^7.19.2",
     "@microsoft/api-extractor": "^7.29.0",
     "@testing-library/react": "^12.1.5",
+    "@testing-library/user-event": "^14.4.2",
+    "@types/babel__core": "^7.1.18",
     "@types/escape-html": "^1.0.2",
```

Comment:
I did not add these babel dependencies myself. They came in when I rebased. I assume I should remove @mkilpatrick ?